src/controllers/taskCommand.ts

```diff
@@ -0,0 +1,58 @@
+import { RDS_BASE_API_URL } from "../constants/urls";
+import { env } from "../typeDefinitions/default.types";
+import { TasksResponseType } from "../typeDefinitions/task.types";
+import { formatTask, generateTaskResponseMessage } from "../utils/formatTask";
+import { discordTextResponse } from "../utils/discordResponse";
+import { getNickName } from "../utils/getNickName";
+import {
+  FAILED_TO_FETCH_TASKS,
+  INVALID_NICKNAME_ERROR,
+  NO_TASKS_FOUND,
+  TASKS_FETCH_FAILED,
+} from "../constants/responses";
+
+async function fetchTasks(assignee: string, status: string) {
+  const url = `${RDS_BASE_API_URL}/tasks?status=${status}&assignee=${assignee}&dev=true`;
+  const response = await fetch(url);
+  const tasks: TasksResponseType = await response.json();
+
+  if (!response.ok) {
+    throw new Error(FAILED_TO_FETCH_TASKS.replace("{{assignee}}", assignee));
+  }
```

Comment:
In what scenario, we may get this error message?

Reply:
When the API failed to fetch tasks.`

Reply:
> When the API failed to fetch tasks.

Okay..what if the user has no tasks to show?

Reply:
https://github.com/Real-Dev-Squad/discord-slash-commands/pull/127/files#diff-12f1a6376ead91fce2084f99f60504079338183fa31571d420caf83ae3714892R34-R37